debian/docker_package.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -aeux
+
+# This should be unique to your repository. Most have a README documenting how to build and produce a binary or
+# other type of artifact. In this case the docker_build.sh script contains npm commands.
+docker run \
+    -i \
+    --rm \
+    -v "$(pwd):/home/go/src/github.com/distil/cfssl" \
+    "./script/build"
+
+# Build the image that will run debuild.
+docker build . -f debian/Dockerfile.debuild -t debuild
+
+# Make the directory with the assets will care about.
+SHORT_HASH=$(git rev-parse --short HEAD)
+VERSION="1~$SHORT_HASH-1"
+
+# This should be unique to your repository.
+DIR="cfssl-$VERSION"
+
+# Insert the version into the changelog.
+sed -i "s/VERSION/$VERSION/g" debian/changelog
```

Comment:
We need to add an extension to the `-i` argument or else it won't work locally. You can them remove the backup files after running `sed`.